### ISSUE TITLE
FSL environment variables

### ISSUE THREAD

**Issue by vinferrer** [ReproNim/neurodocker#256]
Hi guys, I was trying the FSL container and I had to add some environment variables to make it work:
export FSLOUTPUTTYPE=NIFTI_GZ
export FSLMULTIFILEQUIT=TRUE

FSLTCLSH=$FSLDIR/bin/fsltclsh
FSLWISH=$FSLDIR/bin/fslwish

export FSLTCLSH=$FSLDIR/bin/fsltclsh 
export FSLWISH=$FSLDIR/bin/fslwish
FSLLOCKDIR=
FSLMACHINELIST=
FSLREMOTECALL=

export FSLLOCKDIR FSLMACHINELIST FSLREMOTECALL
FSLGECUDAQ=cuda.q

export FSLGECUDAQ

Basically, this is what it is added when the .bashrc script sources $FSLDIR/etc/fslconfig/fsl.sh. I consider that is easier to add as environmental variables in the fsl_recipe. How can I modify fsl.yam template so these variables are included in the recipe?

Thanks in advance,
Cheers

**Comment by kaczmarj** [comment]
@sento4000 - thanks for looking into this. would you like to submit a PR with these changes? you would have to change the file https://github.com/kaczmarj/neurodocker/blob/master/neurodocker/templates/fsl.yaml. you can add these environment variables under the `env` key of that yaml file. instead of using `$FSLDIR`, though, you should use `{{ fsl.install_path }}`, which will fill in the correct `FSLDIR`.

**Comment by djarecka** [comment]
i believed that was solved by #324, if the issues still persist please reopen
